src/web/components/SignInGate/SignInGateSelector.tsx

```diff
@@ -144,6 +144,38 @@ const ShowSignInGate = ({
 	return <></>;
 };
 
+// custom hook can be used by other parts of DCR to determine if the sign in gate will or will not show
+export const useSignInGateWillShow = ({
+	isSignedIn,
+	CAPI,
+}: SignInGateSelectorProps): boolean => {
+	const ab = useAB();
+
+	const test: Runnable | null = ab.firstRunnableTest(tests);
```

Comment:
Does `tests` need to be passed in?

Sorry, by the way, I know this PR is just a draft but I was passing and thought I'd do a drive by review!

Reply:
Yup! The `tests` object in this case relates to the currently live sign in gate tests here: https://github.com/guardian/dotcom-rendering/blob/7c613ec787a9708df96799d2a0203f8d6efe52c1/src/web/components/SignInGate/SignInGateSelector.tsx#L65